modules/cli/src/main/scala/scala/cli/commands/publish/Publish.scala

```diff
@@ -1085,4 +860,214 @@ object Publish extends ScalaCommand[PublishOptions] with BuildCommandHelpers {
         }
     }
   }
+
+  private def buildFileSet(
```

Comment:
Did you make any changes in `buildFileSet` function? I see, that you are new author of this function.
Can you revert this changes to not change git history?